Update the Bar Chart - Auto-Calculated Bar Width demo description (t937471) (#291)
DevExpress/devextreme-demos: JSDemos/Demos/Charts/AutoCalculatedBarWidth/description.md

> When you have multiple [series](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Series_Types/BarSeries/), values for some of their points can be unspecified or set to zero. Bars for these points are not displayed, and space reserved for them is empty.

Reviewer: ```suggestion
When you have multiple [series](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Series_Types/BarSeries/), values for some of their points can be unspecified or set to zero. Bars for these points are not displayed, but space for them remains reserved. This results in gaps.
```

> When you have multiple [series](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Series_Types/BarSeries/), values for some of their points can be unspecified or set to zero. Bars for these points are not displayed, but space for them remains reserved. This results in gaps.

Reviewer 2: "When you have" is not a great way to start a sentence. Technically it's the chart that has series, not "you". And "have" is usually considered a weak and vague verb. 

```suggestion
This demo illustrates a common issue in multi-series charts. Individual [series](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Series_Types/BarSeries/) may skip values for certain arguments or include zero values. If such cases arguments may display different number of bars. This results in uneven gaps between data. 
```

> This demo illustrates a common issue in multi-series charts. Individual [series](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Series_Types/BarSeries/) may skip values for certain arguments or include zero values. If such cases arguments may display different number of bars. This results in uneven gaps between data. 

Reviewer 3: "If such cases arguments may"? 

```suggestion
This demo illustrates a common issue in multi-series charts. Individual [series](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Series_Types/BarSeries/) can skip values for certain arguments or include zero values. In such cases, arguments can display a different number of bars and cause uneven gaps between data. 
```

> The [ignoreEmptyPoints](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Series_Types/BarSeries/#ignoreEmptyPoints) option allows you to prevent this behavior. If you set it to **true**, the gaps will be removed and the bars will populate the empty space as shown in the `Iran`, `Canada`, `Saudi Arabia`, and `Mexico` series.

Reviewer: ```suggestion
The [ignoreEmptyPoints](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Series_Types/BarSeries/#ignoreEmptyPoints) option allows you to prevent this behavior. If you set it to **true**, the neighbouring bars increase their width, thus filling the gaps. In this demo, you can see the option's effect on the example of the `Iran`, `Canada`, `Saudi Arabia`, and `Mexico` arguments.
```

> The [ignoreEmptyPoints](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Series_Types/BarSeries/#ignoreEmptyPoints) option allows you to prevent this behavior. If you set it to **true**, the neighbouring bars increase their width, thus filling the gaps. In this demo, you can see the option's effect on the example of the `Iran`, `Canada`, `Saudi Arabia`, and `Mexico` arguments.

Reviewer 2: I wouldn't say "prevent". It's really not a bad thing and sometimes maybe I'd choose the original behavior. So maybe "change" is a better word. 

I'm not sure what you meant by "neighboring". All these bars are neighboring relative to some other bars :-) 

```suggestion
The [ignoreEmptyPoints](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Series_Types/BarSeries/#ignoreEmptyPoints) option allows you to change this behavior. If you set it to **true**, bars increase their width to fill the gaps. In this demo, you can see how the option affects the following arguments: `Iran`, `Canada`, `Saudi Arabia`, and `Mexico`.
```
